app/models/ems_refresh/save_inventory_physical_infra.rb

```diff
@@ -122,6 +122,14 @@ def save_physical_disk_inventory(parent, hash)
     save_inventory_single(:physical_disk, parent, hash)
   end
 
+  #
+  # Saves the canister information of a storage
+  #
+  def save_canister_inventory(parent, hash)
```

Comment:
If a canister can only belong to a physical storage it'd be better to call this `physical_storage` instead of `parent`.  We use `parent` if something can belong to more than one type of parent.